src/aiperf/accuracy/graders/multiple_choice.py

```
@@ -50,6 +51,10 @@ def _extract_with_flag(self, response_text: str) -> tuple[str, bool]:
         m = _LETTER_RE.search(first_line)
         if m:
             return m.group(1), True
+        # Chain-of-thought output: scan the whole response for the answer.
+        letter, tier = extract_choice_letter(response_text, "ABCD")
```

Ruff (0.15.20)

[warning] Unpacked variable `tier` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)